pkg/operator/resourcesynccontroller/interfaces.go

```diff
@@ -21,3 +31,11 @@ type ResourceSyncer interface {
 	// mirror a deletion from the source.  If the source is a zero object the destination will be deleted.
 	SyncSecret(destination, source ResourceLocation) error
 }
+
+func (rsl *PartialResourceLocation) Equal(other *PartialResourceLocation) bool {
```

Comment:
where do we use this method?

Reply:
I think it's unused now, let me remove it